docs/content/configuration/index.md

```diff
@@ -125,7 +125,7 @@ The Druid servers emit various metrics and alerts via something we call an Emitt
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.emitter`|Setting this value to "noop", "logging", or "http" will instantialize one of the emitter modules.|noop|
+|`druid.emitter`|Setting this value to "noop", "logging", or "http" will instantialize one of the emitter modules. value "composing" can be used to instantialize multiple emitter modules. |noop|
```

Comment:
well... I also thought it was weird. Since the word "instantialize" was already used in the documentation (look at the deleted line), [being a non-native English speaker] I doubted myself and thought may be it was the correct word to use in this situation. :)
will send another PR to fix it.